src/metabase/models/pulse_channel.clj

```diff
@@ -158,18 +158,19 @@
                                       weekday)]
     (k/select PulseChannel
       (k/fields :id :pulse_id :schedule_type :channel_type)
-      (k/where (or (= :schedule_type (name :hourly))
-                   (and (= :schedule_type (name :daily))
-                        (= :schedule_hour hour))
-                   (and (= :schedule_type (name :weekly))
-                        (= :schedule_hour hour)
-                        (= :schedule_day weekday))
-                   (and (= :schedule_type (name :monthly))
-                        (= :schedule_hour hour)
-                        (= :schedule_frame schedule-frame)
-                        (or (= :schedule_day weekday)
-                            ;; this is here specifically to allow for cases where day doesn't have to match
-                            (= :schedule_day monthly-schedule-day-or-nil))))))))
+      (k/where (and (= :enabled true)
+                    (or (= :schedule_type (name :hourly))
+                        (and (= :schedule_type (name :daily))
```

Comment:
I just noticed this, don't know how long it's been this way. But `(name <keyword>)` is just the string name of that keyword. e.g.

```clojure
(name :daily) -> "daily"
```

We should just write the string and save some CPU cycles.

```clojure
(= :schedule_type "monthly")
```